packages/apex-ls/src/server/LCSAdapter.ts

```diff
@@ -381,6 +381,26 @@ export class LCSAdapter {
       },
     );
     this.logger.debug('✅ apex/findMissingArtifact handler registered');
+
+    // Register apexlib/resolve handler for standard library content resolution
+    this.connection.onRequest(
+      'apexlib/resolve',
+      async (params: { uri: string }): Promise<{ content: string }> => {
+        this.logger.debug(
+          `🔍 apexlib/resolve request received for: ${params.uri}`,
+        );
+        try {
+          const { dispatchProcessOnResolve } = await import(
```

Comment:
since we are importing lsp compliant services package in the top, we don't need dynamic importing here anymore, right?